docs/documentation/pages/DECKHOUSE_CONFIGURE2_RU.md

```diff
@@ -0,0 +1,132 @@
+- `<moduleName>Enabled` (где `<moduleName>` — название модуля Deckhouse в camelCase) — позволяет явно включить или отключить модуль. Может принимать значение `"true"` или `"false"` (кавычки обязательны).
+
+> Глобальные настройки и настройки модуля указываются в виде многострочной YAML-строки (строка начинается с символа вертикальной черты — `|`)
+
+Пример ConfigMap `deckhouse`:
+```yaml
+apiVersion: v1
+metadata:
+  name: deckhouse
+  namespace: d8-system
+data:
+  global: |          # Вертикальная черта.
+    # Глобальные настройки в формате YAML.
+    modules:
+      publicDomainTemplate: "%s.kube.company.my"
+  # Настройки модуля kube-dns в формате YAML.
+  kubeDns: |
+    stubZones:
+    - upstreamNameservers:
+      - 192.168.121.55
+      - 10.2.7.80
+      zone: directory.company.my
+    upstreamNameservers:
+    - 10.2.100.55
+    - 10.2.200.55
+  # Отключение модуля dashboard.
+  dashboardEnabled: "false"   
+```
+
+Чтобы изменить конфигурацию Deckhouse отредактируйте ConfigMap `deckhouse`, например следующим способом:
+```shell
+kubectl -n d8-system edit cm/deckhouse
+```
+
+После сохранения конфигурации Deckhouse изменения применяются автоматически. 
+
+## Наборы модулей
+
+Deckhouse работает только с включёнными модулями.
+
+В зависимости от используемого [набора модулей](./modules/020-deckhouse/configuration.html#parameters-bundle) модули могут быть включены или выключены по умолчанию.
+
+{%- assign bundles = site.data.bundles | sort %}
+<table>
+<thead>
+<tr><th>Название набора модулей</th><th>Список включенных по умолчанию модулей</th></tr></thead>
+<tbody>
+{% for bundle in bundles %}
+<tr>
+<td><strong>{{ bundle[0] |  replace_first: "values-", "" | capitalize }}</strong></td>
+<td>{% assign modules = bundle[1] | sort %}
+<ul style="columns: 3">
+{%- for module in modules %}
+{%- assign moduleName = module[0] | regex_replace: "Enabled$", '' | camel_to_snake_case | replace: "_", '-' %}
+{%- assign isExcluded = site.data.exclude.module_names | where: "name", moduleName %}
+{%- if isExcluded.size > 0 %}{% continue %}{% endif %} 
+{%- if module[1] != true %}{% continue %}{% endif %}
+<li>
+{{ module[0] | regex_replace: "Enabled$", '' | camel_to_snake_case | replace: "_", '-' }}</li>
+{%- endfor %}
+</ul>
+</td>
+</tr>
+{%- endfor %}
+</tbody>
+</table>
+
+Выключить модули, включенные по умолчанию в используемом наборе модулей (и наоборот), вы можете в [конфигурации Deckhouse](#конфигурация-deckhouse).
+
+## Выделение узлов под определенный вид нагрузки
+
+Если в параметрах модуля не указаны явные значения `nodeSelector/tolerations`, то для всех модулей используется следующая стратегия:
+
+1. Если параметр `nodeSelector` модуля не указан, то Deckhouse попытается вычислить `nodeSelector` автоматически. В этом случае, если в кластере присутствуют узлы с [лейблами из списка или лейблами определенного формата](#особенности-автоматики-зависящие-от-типа-модуля), то Deckhouse укажет их в качестве `nodeSelector` ресурсам модуля;
```

Comment:
```suggestion
1. Если параметр `nodeSelector` модуля не указан, то Deckhouse попытается вычислить `nodeSelector` автоматически. В этом случае, если в кластере присутствуют узлы с [лейблами из списка или лейблами определенного формата](#особенности-автоматики-зависящие-от-типа-модуля), Deckhouse укажет их в качестве `nodeSelector` ресурсам модуля;
```

I'm not sure about this edit, but it seems to me that the preposition is not needed here.